phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -114,6 +114,15 @@ tokens
     ASYNC='async';
     SAMPLING='sampling';
     UNION='union';
+    FUNCTION='function';
+    AS='as';
+    REPLACE='replace';
```

Comment:
If support for REPLACE won't make this pull request (which is fine b/c it's already big enough), we should just remove it from the grammar for now.